src/tsickle_compiler_host.ts

```diff
@@ -125,6 +126,7 @@ export class TsickleCompilerHost implements ts.CompilerHost {
     }
 
     const sourceFile = this.runConfiguration.oldProgram.getSourceFile(fileName);
+    this.stripAndStoreExistingSourceMap(sourceFile);
```

Comment:
does this need to happen on every getSourceFile? Shouldn't we only do this when loading the file from disk the very first time?

Also: how often does `getSourceFile` get called? We shove megabytes through this, running a regexp over and over again on all inputs could get quite costly.